File: Modul2_C/PRAK304-2510817320010-PutriDeissytaHerliyana.c
#include <stdio.h>

int main() {
    int nilai;
    scanf("%d", &nilai);

    if (nilai == 0) {
        printf("Nol\n");
    } else if (nilai < 10) {
        printf("Satuan\n");
    } else if (nilai < 20) {
        printf("Belasan\n");
    } else if (nilai < 100) {
        printf("Puluhan\n");
    } else {
        printf("Anda Menginput Melebihi Limit Bilangan\n");
    }

    return 0;
}

File: Modul3_Pyton/PRAK304-2510817320010-PutriDeissytaHerliyana.py
nilai = int(input())

if nilai == 0:
    hasil = "Nol"
elif nilai < 10:
    hasil = "Satuan"
elif nilai < 20:
    hasil = "Belasan"
elif nilai < 100:
    hasil = "Puluhan"
else:
    hasil = "Anda Menginput Melebihi Limit Bilangan"

print(hasil)
